src/containers/org/roles/RemoveRoleFromOrgModal.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+
+import { Modal } from 'lattice-ui-kit';
+import { useDispatch } from 'react-redux';
+import type { Organization, Role, UUID } from 'lattice';
+
+import ResetOnUnmount from '../components/ResetOnUnmount';
+import { REMOVE_ROLE_FROM_ORGANIZATION, removeRoleFromOrganization } from '../actions';
+
+
+const resetStatePath = [REMOVE_ROLE_FROM_ORGANIZATION];
+
+type Props = {
+  isVisible :boolean;
+  onClose :() => void;
+  organization :Organization;
+  role :Role;
+};
+
+const RoleDetailsModal = ({
+  isVisible,
+  onClose,
+  organization,
+  role
+} :Props) => {
+  const dispatch = useDispatch();
+  const roleId :UUID = role?.id || '';
+  const organizationId :string = organization?.id || '';
```

Comment:
The modal doesn't render anything specific about the org or role, so it's better to just pass in `organizationId` and `roleId` as props.